x-pack/legacy/plugins/spaces/public/nav_control/nav_control_popover.test.tsx

```diff
@@ -42,6 +42,7 @@ describe('NavControlPopover', () => {
         disabledFeatures: [],
       },
     ]);
+    // @ts-ignore readonly check
```

Comment:
__note:__ looks like we need a separate `ISpaceManager` that all consumers can rely on and then we won't have to cast to `jest.Mocked<SpaceManager>` in `spacesManagerMock.create()` and hence can change properties here without `ts-ignore`. Obviously it's out of scope of this PR, just wanted to note.